docs/user/rstudio/ide/guide/tools/version-control.qmd

```diff
@@ -54,43 +54,43 @@ An excellent resource for learning more about Subversion and how to use it is th
 
 ## Getting Started
 
-Once you've installed your preferred Version Control system, you'll need to activate it on your system by following these steps:
+Once a Version Control system has been installed, it needs to be activated on the system by following these steps:
 
-1.  Go to **Global Options** (from the Tools menu)
+1.  From the **Tools** menu, click **Global Options**
 
-2.  Click **Git/SVN**
+2.  Click on the **Git/SVN** tab
 
 3.  Click **Enable version control interface for RStudio projects**
 
-4.  If necessary, enter the path for your Git or SVN executable where provided. You can also create or add your RSA key for SSH if necessary.
+4.  If necessary, enter the path for the Git or SVN executable where provided. A RSA key can be created or added for SSH if necessary.
```

Comment:
should be An SSH key

We've added support for other types of SSH keys besides RSA and RSA is no longer the default

Reply:
Changed to SSH throughout